config/environment.rb

```diff
@@ -1,5 +1,6 @@
 # Load the rails application
 require File.expand_path('../application', __FILE__)
+require 'system/database'
```

Comment:
I am not sure if this is the proper place of the boot to require this 🤔 Maybe it is, not sure. I am wondering about it because `application` is what requires `rails/all` which requires activerecord 🤔